app/styles/component/_article-featured-video.scss

```diff
@@ -251,4 +215,86 @@
 			}
 		}
 	}
+
+	&.within-portable-infobox {
+		//make sure it is the same as $infobox-background in _portable-infobox.scss
+		$infobox-bg-color: mix(mix($button-color, $black, 50%), $white, 10%);
```

Comment:
can't we extract this variable to be available from both this file and `_portable-infobox.scss`?